src/lib.rs

```diff
@@ -470,6 +470,8 @@ pub struct WebViewAttributes {
   ///
   /// ## Platform-specific:
   ///
+  /// - Windows: Requires WebView2 Runtime version 101.0.1210.39 or higher, does nothing on older versions,
+  /// see https://learn.microsoft.com/en-us/microsoft-edge/webview2/release-notes/archive?tabs=dotnetcsharp#10121039
```

Comment:
I think we should also mention the requirement on `with_incognito` (line 961).

I don't know if the document is enough. Since personal information is quite sensible, the developer may want to know if the mode is enabled or if they must detect it in other ways.

Reply:
Since we documented it requires version `101.0.1210.39` or higher, devs can expect it have no effect on those versions, and provide fallback or something else by comparing the version with `webview_version`